### DISCUSSION TITLE
Does clicking the code viewer button suppose to also open the network inspector dialog?

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#3202] [Q&A] [answered]
As of `1.57.3b1`, clicking on the code viewer button on the logger will also open the network inspector dialog alongside the code viewer. Is this behaviour intended? Thank you.

**Comment by gorhill** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/6e61a13e4065ed327978697afaa1d21de352675c
